docs/01_introduction.md

```diff
@@ -55,23 +53,21 @@ consistent testing of incoming contributions, and thorough regression testing be
 EasyBuild is ***not*** **YABT (Yet Another Build Tool)**: it does not replace established build
 tools like CMake or ``make``, it wraps around them.
 If the installation procedure of a software package involves running some unholy trinity of tools while whispering
-the correct magic incantations, EasyBuild just automates this process so you don't have to go through it manually.
+the correct magic incantations, EasyBuild seeks to automate this process for you.
 
 It is ***not*** **a replacement for traditional Linux package manangers** like ``yum``, ``dnf`` or ``apt``.
-EasyBuild leverages some tools and libraries provided by OS, including glibc, OpenSSL, Infiniband & GPU drivers, etc.,
-so you will still need to use the package management tool that comes with your Linux distribution of choice
-for installing (system) software.
+EasyBuild relies on certain tools and libraries provided by the operating system. This includes glibc, OpenSSL, Infiniband, GPU drivers, and so on. It is required that these tools are installed and managed by other means. This is typically done via the package management tool that comes with your Linux distribution.
 
 Finally, EasyBuild is ***not a magic solution to all your (software installation) problems**.
-* a magic solution to all your (software installation) problems: you may (and will) still run into compiler errors (unless somebody has already taken care of it);
+You may, and probably will still, run into compiler errors unless somebody has already taken care of the problem for you.
```

Comment:
`You may, and probably still will occasionally, run into ...`?

Reply:
Yep.